src/index.js

```diff
@@ -40,25 +40,44 @@
  * @property {string} file.url — image URL
  */
 
-// eslint-disable-next-line
-import css from './index.css';
+import './index.css';
 import Ui from './ui';
 import Tunes from './tunes';
 import ToolboxIcon from './svg/toolbox.svg';
-import Uploader from './uploader';
+
+/**
+ * Interface for classes that represent a uploader
+ * @interface Uploader
+ */
+export class Uploader {
+  /**
+   * Handle clicks on the upload file button
+   * @param {string} url - image source url
+   * @param {Object} context - context object with helpers and other properties
+   * @param {function(string)} context.setPreview - callback for set preview image
+   * @returns {Promise<UploadResponseFormat>}
+   */
+  uploadByUrl(url, { setPreview }) {
```

Comment:
I thought it's better to leave uploader in a package, but give an optional ability to pass the own. It would be simpler for regular users.

Reply:
My goal of this it's reduce main bundle size. If we keep uploader in bundle, total size will be 45kb(like now) than 28kb & 17kb. If we build bundle in es module and umd we can keep uploader in main bundle for cdn users and webpack users.

Reply:
But webpack not support libraryTarger es modules. Only rolllup support it. 
Also in our time, if we build a package for the front-end, then only iife and es modules are needed. Webpack / Rollup work on cheers with esmodules directly, and iife for cdn users

Reply:
With rollup es module have size 30kb (minified)
![image](https://user-images.githubusercontent.com/11390039/55683868-45358500-594d-11e9-92d2-1cd728bf5db9.png)

Reply:
So what?